kolibri_instant_schools_plugin/assets/src/views/sign-in-page/index.vue

```diff
@@ -65,7 +66,27 @@
           :primary="true"
           :disabled="busy"
         />
+
+        <a @click="openPasswordResetModal()" role="link" href="#" id="password-reset">
```

Comment:
Could you remove this block of code and anything else related to password reset? I've implemented this in the PR for pw reset flow?